app/view/index/careerRecords.R

```diff
@@ -185,76 +192,78 @@ server <- function(id) {
             style = "align-items: center; justify-content: space-between;"
           ),
           class = "career-record-button",
-          style = dplyr$if_else(selectedStat == stat, "background-image: linear-gradient(to right, #4b8dad, #e5e5e5 40%);", "")
+          style = dplyr$if_else(
+            selectedStat == stat,
+            "background-image: linear-gradient(to right, #4b8dad, #e5e5e5 40%);",
+            ""
+          )
         )
       }
-      
-      lapply(outstatistics, function(stat){
+
+      lapply(outstatistics, function(stat) {
         output[[paste0(stat, "_record")]] <- shiny$renderUI({
-          outfieldData() |> 
+          outfieldData() |>
             leaderButton(stat = stat, selected = currentStatistic())
         })
       })
-      
-      lapply(keepstatistics, function(stat){
+
+      lapply(keepstatistics, function(stat) {
         output[[paste0(stat, "_record")]] <- shiny$renderUI({
-          keeperData() |> 
+          keeperData() |>
             leaderButton(stat = stat, selected = currentStatisticKeeper())
         })
       })
-      
+
       ## Observers to change the selected statistic for the top 20
-      lapply(outstatistics, function(stat){
+      lapply(outstatistics, function(stat) {
         shiny$observe({
           currentStatistic(stat)
-        }) |> 
+        }) |>
           shiny$bindEvent(
             input[[paste0(stat, "_record_click")]]
           )
       })
-      
-      lapply(keepstatistics, function(stat){
+
+      lapply(keepstatistics, function(stat) {
         shiny$observe(
           currentStatisticKeeper(stat)
-        ) |> 
+        ) |>
           shiny$bindEvent(
             input[[paste0(stat, "_record_click")]]
           )
       })
-      
+
       #### REACTABLE OUTPUT ####
-      top20Reactable <- function(data, stat){
-        data |> 
+      top20Reactable <- function(data, stat) {
+        data |>
           dplyr$select(
             name, club, apps, dplyr$all_of(stat)
-          ) |> 
+          ) |>
           dplyr$arrange(
             dplyr$across(
               dplyr$starts_with(stat),
               dplyr$desc
             )
-          ) |> 
-          dplyr$slice_head(n = 20) |> 
+          ) |>
+          dplyr$slice_head(n = 20) |>
           dplyr$mutate(
-            RANK = 1:dplyr$n()
-          ) |> 
-          dplyr$relocate(RANK) |> 
+            RANK = seq_len(dplyr$n())
```

Comment:
Not sure if this can mess with any of our stuff, but `seq_len` was recommended by the linter as a more stable option for any edge cases.